### PR TITLE
My Great Big Fat Heretic Proofreading

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Erratas almost the entire list of Heretic descriptions. Most are grammar fixes, but some required more extensive rewrites to correct strange sentence structure. Requesting @EdgeLordExe to give feedback on this, with Heretic being his baby and all that.

## Why It's Good For The Game
image
(The descriptions are a really good base, but they needed some work.)

## Changelog
:cl:
tweak: Heretic has received a minor textual facelift.
/:cl:

### CODE REVIEW

**Comment by EdgeLordExe** [comment]
I love it

File: code/modules/antagonists/eldritch_cult/eldritch_items.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 /datum/action/innate/heretic_shatter
 	name = "Shattering Offer"
-	desc = "By breaking your blade you are noticed by the hill or rust and are granted an escape from a dire sitatuion. (Teleports you to a random safe z turf on your current z level but destroys your blade.)"
+	desc = "By breaking your blade you are noticed by the Rusted Hills and are granted salvation from a dire sitatuion. (Teleports you to a random safe turf on your current z level, but destroys your blade.)"
```
Comment (Jared-Fogle) [inline code/modules/antagonists/eldritch_cult/eldritch_items.dm:37]:
```suggestion
	desc = "By breaking your blade you are noticed by the Rusted Hills and are granted salvation from a dire situation. (Teleports you to a random safe turf on your current z level, but destroys your blade.)"
```

File: code/modules/antagonists/eldritch_cult/eldritch_items.dm [hunk 2]
```diff
@@ -57,12 +57,12 @@
 /datum/action/innate/heretic_shatter/Activate()
 	var/turf/safe_turf = find_safe_turf(zlevels = sword.z, extended_safety_checks = TRUE)
 	do_teleport(holder,safe_turf,forceMove = TRUE)
-	to_chat(holder,"<span class='warning'> You feel a gust of energy flow through your body, Rusted Hills heard your call...")
+	to_chat(holder,"<span class='warning'> You feel a gust of energy flow through your body... the Rusted Hills heard your call...")
```
Comment (Jared-Fogle) [inline code/modules/antagonists/eldritch_cult/eldritch_items.dm:60]:
```suggestion
	to_chat(holder,"<span class='warning'>You feel a gust of energy flow through your body... the Rusted Hills heard your call...</span>")
```

**Comment by EdgeLordExe** [comment]
@EOBGames hello?

**Comment by Jared-Fogle** [comment]
if lonely 3am nights cause typos then we'll need to start the free coder girlfriends fund, or better alarm clocks

**Comment by EOBGames** [comment]
> if lonely 3am nights cause typos then we'll need to start the free coder girlfriends fund, or better alarm clocks

I'll take a better alarm clock, I keep sleeping through mine

**Pull request merged**